src/components/Dashboard/Dashboard.tsx

```diff
@@ -1,4 +1,4 @@
-export default function Dashboard(){
```

Comment:
please change de file name to dashboard without upper case, and change the function name to Dashboard with upper case

Reply:
Done